managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx

```diff
@@ -417,7 +417,7 @@ To complete the following procedure, you must have:
 
 To complete the following procedure, you must have:
 
-- Installed PostgreSQL 16 (or newer) with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
+- Installed PostgreSQL 15 (or older) with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
```

Comment:
Installed PostgreSQL in a version compatible with your target Managed PostgreSQL, with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).

Reply:
```suggestion
- Installed PostgreSQL in a version compatible with your target Managed PostgreSQL, with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
```